Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1455,14 +1463,6 @@ bool RenderBox::hasTrimmedMargin(PhysicalDirection physicalDirection) const
 {
     if (!isInFlow())
         return false;
-#if ASSERT_ENABLED
```

Comment:
This seems like it could just be a separate commit for code cleanup...

Reply:
Sounds good to me. Added the code block back